lib/IRGen/IRGenSIL.cpp

```diff
@@ -2074,31 +2098,69 @@ void IRGenSILFunction::visitGraphOperationInst(GraphOperationInst *i) {
       assert(0 && "TensorAttributes cannot be dynamic");
     case GraphOperationInfo::ArgumentLowering::Input: {
       assert(argumentName.empty() && "inputs cannot have names");
+
+      // Special attr to represent the list length, if needed. This is necessary
+      // because we add list inputs one element at a time rather than by using a
+      // TFE_OpAddInputList function (which doesn't exist yet). This is null
+      // when the input does not have a number attr.
+      const char *numberAttr = inputListNumberAttr(opInfo.getOperationName(),
```

Comment:
is there an op that can take 2 or more input lists? say the op takes total of 3 input params, and they are (list, handle, list). 

if yes:
1. in that case, would there be two "numberAttr" attrs, maybe named "N1" and "N2"?
2. i guess the code here wouldn't generalize to those cases (if so, pls feel free to address later -- this patch is already very large).

Reply:
Yes, there are such ops. For example, the "ParseExample" example that I added in the comments. The code should already handle such cases (as long as we add the ops to the hardcoded database in `getInputListNumberAttr`).